### PR TITLE
Update configurations and instructions

### PR DESCRIPTION
**Pull request by razvand**
Update configurations and instructions and Make-based. Include multiple architectures, compilers, VMMs.

### CODE REVIEW

**Review by StefanJum — CHANGES_REQUESTED** [review]
@razvand in the README file, change all occurrences of `.config.redis-qemu-x86_64` to `.config.redis-qemu-x86_64-9pfs`. Same for `aarch64` and for the running scripts (for everything that you renamed, basically).

**Comment by razvand** [comment]
> @razvand in the README file, change all occurrences of `.config.redis-qemu-x86_64` to `.config.redis-qemu-x86_64-9pfs`. Same for `aarch64` and for the running scripts (for everything that you renamed, basically).

@StefanJum , done.

File: README.md [hunk 1]
```diff
@@ -18,9 +74,9 @@ git clone https://github.com/unikraft/unikraft .unikraft/unikraft
 git clone https://github.com/unikraft/lib-redis .unikraft/libs/redis
 git clone https://github.com/unikraft/lib-musl .unikraft/libs/musl
 git clone https://github.com/unikraft/lib-lwip .unikraft/libs/lwip
-UK_DEFCONFIG=$(pwd)/.config.redis_qemu-x86_64 make defconfig
+UK_DEFCONFIG=$(pwd)/.config.redis_qemu-x86_64-9pfs make defconfig
```
Comment (RaduNichita) [inline README.md]:
```suggestion
UK_DEFCONFIG=$(pwd)/.config.redis-qemu-x86_64-9pfs make defconfig
```

File: README.md [hunk 2]
```diff
@@ -29,15 +85,15 @@ The same can be done for `AArch64`, by running the commands below:
 
 ```console
 make properclean
-UK_DEFCONFIG=$(pwd)/.config.redis_qemu-arm64 make defconfig
+UK_DEFCONFIG=$(pwd)/.config.redis-qemu-arm64-9pfs make defconfig
```
Comment (RaduNichita) [inline README.md]:
```suggestion
UK_DEFCONFIG=$(pwd)/.config.redis-qemu-aarch64-9pfs make defconfig
```

File: README.md [hunk 3]
```diff
@@ -117,7 +173,7 @@ Follow the steps below for the setup:
       This will print the contents of the repository.
 
      ```text
-     fs0/  kraft.yaml  Makefile  Makefile.uk.default  README.md .config.redis_qemu-x86_64 .config.redis_qemu-arm64
+     fs0/  kraft.yaml  Makefile  Makefile.uk.default  README.md .config.redis-qemu-x86_64-9pfs .config.redis_qemu-arm64-9pfs [...]  run-fc-x86_64-initrd.sh*  run-qemu-aarch64-9pfs.sh*  [...]
```
Comment (RaduNichita) [inline README.md]:
```suggestion
     fs0/  kraft.yaml  Makefile  Makefile.uk.default  README.md .config.redis-qemu-x86_64-9pfs .config.redis_qemu-aarch64-9pfs [...]  run-fc-x86_64-initrd.sh*  run-qemu-aarch64-9pfs.sh*  [...]
```

**Review by RaduNichita — CHANGES_REQUESTED** [review]
Looks good. 

Please replace the occurences of `arm64` with `aarch64` in the README file.

**Review by StefanJum — APPROVED** [review]
All good on my side, I'll add the tag after @RaduNichita is happy.

**Review by RaduNichita — APPROVED** [review]
All good with me as well.

Reviewed-by: Radu Nichita <radunichita99@gmail.com>

**Review by StefanJum — APPROVED** [review]
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request merged**